config/development.json

```diff
@@ -1,4 +1,10 @@
 {
     "jwtPrivateKey": "epic",
-    "database" : "devDb"
+    "database" : {
```

Comment:
Please remove all database config from the repo. You can use connection strings in the `.env` file.